packages/atlas/lib/src/screen_coordinate.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/foundation.dart';
+
+/// A pair of x and y coordinates.
+/// The `x` and `y` coordinates are stored as screen pixels (not display pixels)
+/// relative to the top of the map, not top left of the whole screen.
+class ScreenCoordinate {

```

Comment:
I feel like this class should be called `ScreenCoordinates` instead of `ScreenCoordinate`
@kmjaris @noahmateen do you agree?